config/routes.rb

```diff
@@ -2,6 +2,8 @@
   devise_for :users
   root 'welcome#index'
 
+  get 'about' => 'welcome#about', as: :about
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.